src/core/server/config/deprecation/core_deprecations.test.ts

```diff
@@ -22,53 +22,53 @@ describe('core deprecations', () => {
     it('logs a warning if CONFIG_PATH environ variable is set', () => {
       process.env.CONFIG_PATH = 'somepath';
       const { messages } = applyCoreDeprecations();
-      expect(messages).toMatchInlineSnapshot(`
-        Array [
-          "Environment variable \\"CONFIG_PATH\\" is deprecated. It has been replaced with \\"KBN_PATH_CONF\\" pointing to a config folder",
-        ]
-      `);
+      expect(messages).toEqual(
+        expect.arrayContaining([
+          'Environment variable "CONFIG_PATH" is deprecated. It has been replaced with "KBN_PATH_CONF" pointing to a config folder',
+        ])
+      );
```

Comment:
Adding a deprecation that's always registered broke all of our snapshot tests, as they were taking a snapshot of the entire list of deprecation messages. So rather than bloat the snapshots by inlining the new deprecation in every test, I changed the assertions to use `expect.arrayContaining` instead.